src/Features/CSharp/Portable/CodeRefactorings/ConvertConversionOperators/ConvertConversionOperatorsFromAsRefactoringProvider.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CodeRefactorings.ConvertConversionOperators;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.ConvertConversionOperators
+{
+    /// <summary>
+    /// Refactor:
+    ///     var o = 1 as object;
+    ///
+    /// Into:
+    ///     var o = (object)1;
+    /// </summary>
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = PredefinedCodeRefactoringProviderNames.ConvertConversionOperators), Shared]
```

Comment:
I use `PredefinedCodeRefactoringProviderNames.ConvertConversionOperators` on all 4 providers. Is this okay?

Reply:
probably not.  we don't want two providers from the same language having the same name.  we should have two names here, one for each direction.  sorry!

Reply:
Done 46ca1b68